src/components/Modal/ConfirmModal/index.tsx

```diff
@@ -95,9 +120,9 @@ export const ConfirmModal = ({
       bodyScrollLockTarget={bodyScrollLockTarget}
     >
       <Container mobileWidth={mobileWidth} mobileHeight={mobileHeight}>
-        <Body id={modalId && `${modalType}${modalId}`}>
+        <Body id={modalId ? `${modalType}${modalId}` : undefined}>
           {icon && <Icon src={getIconSrc(icon)} alt="icon" />}
-          {title && <TitleText hasIcon={icon}>{title}</TitleText>}
+          {title && <TitleText hasIcon={!!icon}>{title}</TitleText>}
```

Comment:
```suggestion
          {title && <TitleText hasIcon={icon}>{title}</TitleText>}
```
icon도 항상 success가 기본으로 되어있는거같은데 없어도 괜찮지 않을까유?

Reply:
요것은 hasIcon의 type은 boolean이고 icon의 type은 string이기 때문에 형변환 해준 것입니다!